R/getCrossAssociation.R

```diff
@@ -367,6 +370,7 @@ setMethod("getCrossAssociation", signature = "SummarizedExperiment",
                                               altexp2 = NULL,
                                               col.var1 = NULL,
                                               col.var2 = NULL,
+                                              by = MARGIN,
```

Comment:
This is internal function so no need to keep MARGIN here anymore

Reply:
This internal function is used only from getCrossAssociation which already takes this name change into account